src/Meshes/Meshes.jl

```diff
@@ -164,6 +170,34 @@ function Mesh(; nodes::Vector{N}=Vector{AbstractNode}(),
     Mesh(nodes, elements, faces, node_sets, element_sets, face_sets, extra)
 end
 
+"Return the mesh node coordinates matrix. Each row is a node, each column a coordinate."
+function node_coordinates_matrix(mesh::Mesh{dim,T}) where {dim,T}
+    nodes_coords_matrix = Matrix{Float64}(undef, (num_nodes(mesh), dim))
+    for (i, n) in enumerate(nodes(mesh))
+        nodes_coords_matrix[i, :] = coordinates(n)
```

Comment:
any reason to define it this way and not as its transposed? it sounds more natural to me if coordinates of each node ara arranged by columns (it also fits more natural with Julia's column-based indexing).

Reply:
Done, this is compatible with WriteVTK